tee-worker/omni-executor/rpc-server/src/methods/pumpx/export_wallet.rs

```diff
@@ -14,6 +14,7 @@ use sha2::Sha256;
 
 #[derive(Debug, Deserialize)]
 pub struct ExportWalletParams {
+	pub user_uid: String,
```

Comment:
How do we make sure both `user_id` and `user_email` belong to same user ?

Reply:
Good point, we'll need to query the uid for an email from backend and verify it 😂  let me adjust it

Reply:
I read this after my comment 😆